implementations/micrometer-registry-azure/src/main/java/io/micrometer/azure/AzureMeterRegistry.java

```diff
@@ -0,0 +1,318 @@
+     */
+    private void trackLongTaskTimer(Meter.Id id, Map<String, String> properties,
+        LongTaskTimer meter) {
+        MetricTelemetry metricTelemetry = createAndGetBareBoneMetricTelemetry(getConventionName(id, "active"), properties);
+        metricTelemetry.setValue(meter.activeTasks());
+        client.trackMetric(metricTelemetry);
+        logger.trace(String.format("sent LongTaskTimer metric with name %s", metricTelemetry.getName()));
+
+        metricTelemetry = createAndGetBareBoneMetricTelemetry(getConventionName(id, "duration"), properties);
+        metricTelemetry.setValue(meter.duration(getBaseTimeUnit()));
+        client.trackMetric(metricTelemetry);
+        logger.trace(String.format("sent LongTaskTimer metric with name %s", metricTelemetry.getName()));
+    }
+
+    /**
+     * Converts DistributionSummary type meter to Azure format and transmits to Azure Monitor backend
+     * @param meterName name of the DistributionSummary meter
+     * @param properties dimensions of the metric
+     * @param meter Meter holding rate aggregated metric values
+     */
+    private void trackDistributionSummary(String meterName, Map<String, String> properties,
+        DistributionSummary meter) {
+        MetricTelemetry metricTelemetry = createAndGetBareBoneMetricTelemetry(meterName, properties);
+        metricTelemetry.setValue(meter.totalAmount());
+        metricTelemetry.setCount((int)meter.count());
+        metricTelemetry.setMax(meter.max());
+
+        // TODO: Is setting min to 0th percentile value apt workaround?
+        HistogramSnapshot snapshot = meter.takeSnapshot();
+        Optional<ValueAtPercentile[]> opt = Optional.ofNullable(snapshot.percentileValues());
+        opt.ifPresent(u -> { if (u.length > 0) metricTelemetry.setMin(u[0].value(TimeUnit.MILLISECONDS)); });
+        client.trackMetric(metricTelemetry);
+        logger.trace(String.format("sent DistributionSummary metric with name %s", metricTelemetry.getName()));
+
+    }
+
+    /**
+     * Converts Timer type meter to Azure Format and transmits to Azure Monitor backend
+     * @param meterName Name of the Timer Meter
+     * @param properties Dimensions of the metric
+     * @param meter Timer type Meter holding rate aggregated metric values
+     */
+    private void trackTimer(String meterName, Map<String, String> properties, Timer meter) {
+        MetricTelemetry metricTelemetry = createAndGetBareBoneMetricTelemetry(meterName, properties);
+        metricTelemetry.setValue(meter.totalTime(getBaseTimeUnit()));
+        metricTelemetry.setCount((int)meter.count());
+
+        // TODO: Is setting min to 0th percentile value apt workaround?
+        HistogramSnapshot snapshot = meter.takeSnapshot();
+        Optional<ValueAtPercentile[]> opt = Optional.ofNullable(snapshot.percentileValues());
+        opt.ifPresent(u -> { if (u.length > 0) metricTelemetry.setMin(u[0].value(TimeUnit.MILLISECONDS)); });
```

Comment:
@macrogreg it should be noted that min is not always emitted by Micrometer and it depends if there is percentile snapshot available. Do we want to keep this behavior to emit 0th percentile in place of min when available?